tests/integration/devfile/cmd_devfile_deploy_test.go

```diff
@@ -28,7 +28,6 @@ var _ = Describe("odo devfile deploy command tests", func() {
 		BeforeEach(func() {
 			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
 			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-deploy.yaml"), path.Join(commonVar.Context, "devfile.yaml"))
-			helper.Cmd("odo", "create").ShouldPass()
```

Comment:
You'll need to remove this from https://github.com/redhat-developer/odo/blob/main/tests/integration/devfile/cmd_devfile_delete_test.go#L43 as well.

Reply:
Are you sure? In the test, we are using the create component to test `odo delete -a` or `odo delete`

Reply:
Oh right, my bad. You can move the tests(https://github.com/redhat-developer/odo/blob/main/tests/integration/devfile/cmd_devfile_delete_test.go#L49) to a different block in that case.

Reply:
I don't think so, as we want to test the different `odo delete` cases (only pushed, only deployed, or both) in this block, so we need to have `push`ed and `deploy`ed.